changelog/28062.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:change
```

Comment:
I think this should be `improvement`? `change` is usually reserved for breaking changes or things that might affect an existing workflow.

Reply:
good point, updated